fission/util/version.go

```diff
@@ -0,0 +1,48 @@
+package util
```

Comment:
why is this in util?  It could just be fission/version.go, right?

Reply:
This file is only used in CLI and was put here due to import cycle problem.